src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/KinesisShardConsumer.java

```diff
@@ -64,24 +61,36 @@ class ShardConsumer {
     private final long taskBackoffTimeMillis;
     private final boolean skipShardSyncAtWorkerInitializationIfLeasesExist;
 
-    @Getter
+    //@Getter
```

Comment:
why are we commenting it out and adding separate get function? same for below two commented gets

Reply:
This is a carry over from the brazil testing as the build couldn't find the getter functions. I will verify that maven can properly read the getter, and remove these changes if tests pass.